ProteoformSuiteGUI/ProteoformFamilies.cs

```diff
@@ -134,54 +151,60 @@ public void update_figures_of_merit()
 
         private void cmbx_tableSelector_SelectedIndexChanged(object sender, EventArgs e)
         {
-            if (cmbx_tableSelector.SelectedIndex == 0) fill_proteoform_families(tb_tableFilter.Text);
+            if (cmbx_tableSelector.SelectedIndex == 0) fill_proteoform_families(tb_tableFilter.Text, -1);
             else if (cmbx_tableSelector.SelectedIndex == 1) fill_theoreticals(tb_tableFilter.Text);
             else if (cmbx_tableSelector.SelectedIndex == 2) fill_go(Aspect.BiologicalProcess, tb_tableFilter.Text);
             else if (cmbx_tableSelector.SelectedIndex == 3) fill_go(Aspect.CellularComponent, tb_tableFilter.Text);
             else if (cmbx_tableSelector.SelectedIndex == 4) fill_go(Aspect.MolecularFunction, tb_tableFilter.Text);
+            else if (cmbx_tableSelector.SelectedIndex > 4) fill_proteoform_families(tb_tableFilter.Text, cmbx_tableSelector.SelectedIndex - 5);
         }
 
         private void tb_tableFilter_TextChanged(object sender, EventArgs e)
         {
-            if (cmbx_tableSelector.SelectedIndex == 0) fill_proteoform_families(tb_tableFilter.Text);
+            if (cmbx_tableSelector.SelectedIndex == 0) fill_proteoform_families(tb_tableFilter.Text, -1);
             else if (cmbx_tableSelector.SelectedIndex == 1) fill_theoreticals(tb_tableFilter.Text);
             else if (cmbx_tableSelector.SelectedIndex == 2) fill_go(Aspect.BiologicalProcess, tb_tableFilter.Text);
             else if (cmbx_tableSelector.SelectedIndex == 3) fill_go(Aspect.CellularComponent, tb_tableFilter.Text);
+            else if (cmbx_tableSelector.SelectedIndex > 4) fill_proteoform_families(tb_tableFilter.Text, cmbx_tableSelector.SelectedIndex - 5);
         }
 
-        public void fill_proteoform_families(string filter)
+        public void fill_proteoform_families(string filter, int decoyCommunityMinusOneIsTarget)
```

Comment:
For now at least while still testing, I think it's helpful to have the option to fill the dgv with decoy communities, so that int is to distinguish which decoy community - if the int is 0 or above, it fills with decoy_community_0 or whichever #. If that int is set to -1, it fills the dgv with the target community families